hw/mcu/dialog/da1469x/include/mcu/da1469x_dma.h

```diff
@@ -43,15 +43,24 @@ extern "C" {
 #define MCU_DMA_PERIPH_SDADC            (13)
 #define MCU_DMA_PERIPH_NONE             (15)
 
+/* DMA burst mode can be one of following values */
```

Comment:
Those new #defines are already in DA1469xAB.h

Those that were removed were specially made for configuration structure that is later used for register writes.

So in short I would suggest not to change this file at all, and use existing defines in otp source.

Reply:
As explained above, I found the #defines to be useless unless we wanted to add helper functions (inline).  So we'd have the following:
- Additional values for the actual contents of the fields we are setting
- Inline functions to set / clear / extract values